aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -194,6 +206,10 @@ Tensor sub(const Tensor& self, Scalar other, Scalar alpha) {
   return native::sub(self, wrapped_scalar_tensor(other), alpha);
 }
 
+Tensor sub(Scalar other, const Tensor& self, Scalar alpha) {
+  return native::sub(wrapped_scalar_tensor(other), self, alpha);
```

Comment:
this is wrong, same as add.

Reply:
I think this is correct? It is different from `add`.